### PR TITLE
test: add useVirtualList test case 

### PR DESCRIPTION
**Pull request by riyueweiyi**
[[中文版模板 / Chinese template](https://github.com/alibaba/hooks/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: packages/hooks/src/useSize/index.ts [hunk 1]
```diff
@@ -7,7 +7,13 @@ import useIsomorphicLayoutEffectWithTarget from '../utils/useIsomorphicLayoutEff
 type Size = { width: number; height: number };
 
 function useSize(target: BasicTarget): Size | undefined {
-  const [state, setState] = useRafState<Size>();
+  const [state, setState] = useRafState<Size>(() => {
+    const el = getTargetElement(target);
+    return {
```
Comment (brickspert) [inline packages/hooks/src/useSize/index.ts]: 这个不能改，这里设计的就是没有值时为 undefined 的。

File: packages/hooks/src/useVirtualList/index.en-US.md [hunk 2]
```diff
@@ -23,8 +23,8 @@ A hook that allows you to use virtual list to render huge chunks of list data.
 const [list, scrollTo]= useVirtualList<T>(
   originalList: T[],
   options: {
-    containerTarget: (() => Element) | Element | MutableRefObject<Element>,
```
Comment (brickspert) [inline packages/hooks/src/useVirtualList/index.en-US.md]: 这里也不能改，这里是支持 Element 的

File: packages/hooks/src/useSize/index.ts (hunk 1 above)
Comment (riyueweiyi) [reply packages/hooks/src/useSize/index.ts]: 这里可以去掉

File: packages/hooks/src/useVirtualList/index.en-US.md (hunk 2 above)
Comment (riyueweiyi) [reply packages/hooks/src/useVirtualList/index.en-US.md]: 这里要支持Element需要改动代码逻辑

Comment (brickspert) [reply packages/hooks/src/useVirtualList/index.en-US.md]: 应该不需要的吧，都是统一处理的。

**Pull request closed without merging**